src/main/kotlin/no/nav/familie/ef/sak/mapper/IverksettingDtoMapper.kt

```diff
@@ -0,0 +1,165 @@
+            VedtaksdetaljerDto(resultat = vedtak.resultatType.tilVedtaksresultat(),
+                               vedtaksdato = LocalDate.now(), // TODO: Er dette når første saksbehandler fullfører eller når beslutter godkjenner
+                               opphørÅrsak = null, // TODO: Revurdering
+                               saksbehandlerId = saksbehandler,
+                               beslutterId = beslutter,
+                               tilkjentYtelse = TilkjentYtelseDto(
+                                       andelerTilkjentYtelse = tilkjentYtelse.andelerTilkjentYtelse.map { andel ->
+                                           AndelTilkjentYtelseDto(periodebeløp = PeriodebeløpDto(beløp = andel.beløp,
+                                                                                                 periodetype = Periodetype.MÅNED,
+                                                                                                 fraOgMed = andel.stønadFom,
+                                                                                                 tilOgMed = andel.stønadTom),
+                                                                  kildeBehandlingId = andel.kildeBehandlingId)
+                                       }
+
+                               ),
+                               inntekter = vedtak.inntekter?.inntekter?.map {
+                                   InntektDto(periodebeløp = PeriodebeløpDto(beløp = it.inntekt.intValueExact(),
+                                                                             periodetype = Periodetype.MÅNED,
+                                                                             fraOgMed = it.startDato,
+                                                                             tilOgMed = it.sluttDato),
+                                              inntektstype = InntektsType.ARBEIDINNTEKT) // TODO: Hva er inntektstype?
+                               } ?: emptyList()
+            )
+
+    private fun mapSøkerDto(fagsak: Fagsak, behandling: Behandling): SøkerDto {
+        val søknad = søknadService.hentOvergangsstønad(behandling.id)
+        val ( grunnlagsdata )  = persisterGrunnlagsdataService.hentGrunnlagsdata(behandling.id)
+        val alleBarn = BarnMatcher.kobleSøknadsbarnOgRegisterBarn(søknad.barn, grunnlagsdata.barn)
+        val navEnhet = arbeidsfordelingService.hentNavEnhetIdEllerBrukMaskinellEnhetHvisNull(søknad.fødselsnummer)
+        val harSagtOpp = hentHarSagtOppEllerRedusertFraVurderinger(behandling)
+
+        return SøkerDto(kode6eller7 = grunnlagsdata.søker.adressebeskyttelse?.let {
+            listOf(AdressebeskyttelseGradering.STRENGT_FORTROLIG,
+                   AdressebeskyttelseGradering.STRENGT_FORTROLIG_UTLAND,
+                   AdressebeskyttelseGradering.FORTROLIG).contains(it.gradering)
+        }
+                                      ?: false,
+                        personIdent = fagsak.hentAktivIdent(),
```

Comment:
Fint hvis dere oppdaterer til siste versjon av kontrakter og legger til `allePersonIdenter` (då vi ikke har alle personIdenter ennå i fagsaken og oppdaterer disse kan det være fint å bruke kallet mot integrasjoner som henter alle)

Reply:
vi bruker bare de som ligger i fagsak, men bør vi gjøre kall mot integrasjoner isteden?

Reply:
Et annet poeng; burde ef-iverksett gjøre et kall selv mot integrasjoner for å hente alle identene?

Reply:
PersonIdenter i fagsak blir aldri oppdatert.
Ja, det er en god poeng. Det med alle identer er ju bare noe midlertidig når vi sender hendelser till feed. Men de feedhendelsene skal vel fjernes sen.